x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/use_ilm_explain/index.test.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { renderHook } from '@testing-library/react-hooks';
+import React from 'react';
+
+import { DataQualityProvider } from '../data_quality_panel/data_quality_context';
+import { mockIlmExplain } from '../mock/ilm_explain/mock_ilm_explain';
+import { ERROR_LOADING_ILM_EXPLAIN } from '../translations';
+import { useIlmExplain } from '.';
+
+const mockHttpFetch = jest.fn();
+const ContextWrapper: React.FC = ({ children }) => (
+  <DataQualityProvider httpFetch={mockHttpFetch}>{children}</DataQualityProvider>
+);
+
+const pattern = 'packetbeat-*';
+
+describe('useIlmExplain', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('happy path', () => {
```

Comment:
nit: If you do end up adding another commit, can you re-label this to `successful response from ilm endpoint` or something.. This is fine too, just not the most descriptive 😄